time_weighted_average_price/attest_fn_call/Makefile

```diff
@@ -37,34 +37,36 @@ build: tmp/x.wasm
 
 .PHONY: prev
 prev: check
-	$(eval prev_ea := $(shell jq '.enclave_attested_application_public_key.enclave_attestation' tmp/prev.json | sed 's/\//\\\//g' ))
-	$(eval prev_ta := $(shell jq '.transitive_attested_function_call.transitive_attestation' tmp/prev.json | sed 's/\//\\\//g' ))
+	$(eval prev_ea := $(shell jq -r '.enclave_attested_application_public_key.enclave_attestation' tmp/prev.json))
+	$(eval prev_ta := $(shell jq -r '.transitive_attested_function_call.transitive_attestation' tmp/prev.json ))
```

Comment:
some small updates to the make recipes since I am replicating them in the testscript and these seemed like they were doing some substitutions instead of using a different sed delimiter, which I think reduces the number of steps that can error and helps the reader follow more easily.

Reply:
my understanding is this was escaping / characters because / is allowed in base64 encoding. I changed the sed delimiter below to | instead so all base64 characters are safe and we don't need to do any defensive escaping.